### ISSUE TITLE
SQLite Manager does not support importing pipe delimited files

### ISSUE THREAD

**Issue by andrewpuleo** [tthibo/SQL-Tutorial#11]
# *SQLite Manager* Problems 😬 
## Description 📝 
Upon starting the tutorial I noticed that the link to download the *SQLite Manager* add-on for Firefox was broken. 
* Here is an updated link: [Firefox add-on](https://addons.mozilla.org/en-US/firefox/addon/sqlite-manager-webext/). 
* Alternatively, here is a link for the Chrome browser extension: [Chrome add-on](https://chrome.google.com/webstore/detail/sqlite-manager/njognipnngillknkhikjecpnbkefclfe?hl=en)
Then I encountered another issue; *SQLite Manager* cannot import from files delimited by pipes ("|").
<img width="1089" alt="screen shot 2018-12-24 at 11 19 10 pm" src="https://user-images.githubusercontent.com/26037941/50415885-645f3b00-07d2-11e9-94bf-2fffba6396c9.png">   

[Source](https://add0n.com/sqlite-manager.html)   

_**Note:**_ I never saw an *Import Wizard* icon either, as shown in the tutorial. And only after reading the above source did I experiment with dragging and dropping a csv file into the browser.

## Solution ✅ 
I ended up downloading [SQLite Browser](https://sqlitebrowser.org/) and using it for the tutorial. It is easier to use imo and has more people actively contributing to it :+1:    

I liked this tutorial a lot. Perhaps it could be made more user friendly by either
1. Updating the datasets to be CSV formatted,
2. Or adapting the steps to use *SQLite Browser*.

<img width="1276" alt="screen shot 2018-12-24 at 11 29 41 pm" src="https://user-images.githubusercontent.com/26037941/50416139-dbe19a00-07d3-11e9-8559-3f04d5c06afa.png">

**Comment by tthibo** [comment]
Hi Andrew -- Thanks very much for this feedback. The newest version of this tutorial is now being hosted [here](https://a-gentle-introduction-to-sql.readthedocs.io/en/latest/) and is being maintained by @zstumgoren. I need to update the README to point to that page. Please take a look and let Serdar or me know if your issues aren't addressed there. Thanks!

**Comment by andrewpuleo** [comment]
Hi Troy, thanks for the quick response! The tutorial that you linked to does take care of the issue :+1: Thanks again,
Andrew

**Comment by tthibo** [comment]
Great!
